uniffi/src/bindings/python/templates/ObjectTemplate.py

```diff
@@ -6,7 +6,8 @@ def __init__(self, {% call py::arg_list_decl(cons.arguments()) -%}):
         self._handle = {% call py::to_rs_call(cons) %}
     {%- endfor %}
 
-    # XXX TODO: destructors or equivalent.
+    def __del__(self):
```

Comment:
Should this similarly poison the object on the python side, like we do in Kotlin? IIRC it is possible that someone might subclass one of these and extend `__del__` in a way that resurrects the the python-side object after the rust-side destructor has been called. (They really *shouldn't* do that, but it's *possible*).

Reply:
Ug. I think I'm going to file a python-only issue to investigate stopping resurrecting python side objects.